### ISSUE TITLE
Mostro should allow open disputes only after the seller pays the hold invoice 

### ISSUE THREAD

**Issue by Catrya** [MostroP2P/mostro#231]
bug: mostro lets the seller open a dispute just by taking an order, without even having to pay the invoice.

i think mostro should only allow open a new dispute after the seller pays the hold invoice and the buyer submits their invoice

This is what mostro do:

```
Mostro sent you this message for order id: 6d30e4eb-7ca0-4f81-9c3e-3da6ac262d8d at 20:00 date - 26/03/2024

Pay this invoice to continue --> lnbcrt5u1pnqx2tnpp5fsu9et982nhew6ffxpk86s68dtfm56sk9wur4jsfssfv0fwskddqdvldec82c33w9chzut30pu8qdrrvv6nqdm8wpcxgamgv3exk7nt09envet4v3urqafnvd5rvwr6dp58gatjxuc8wdm40fenvvnew3jhqgpdypzhxcmjdamjqctdda6kuapqfaexgetjyq3nvepnxpjngetz95mkxcfs956xvwp395ukxvm995ekgcfkv93nyd3jvsuxgw3q2dz5cnpqgf2yxgrxdaezqs642qsrvvpq95syjapq2ay5cnpqgefy2326g5syjn3q2aq5cnz92shzqjt5ypmkjmrvypex2mr9v9ek2gr0de3k2greda6jqun9d3jkzum99csyjapqwa5kcmpqwfjhgatjdcsxje3qvf6hjetjypjx7etnyphx7apqvdhkuenfwfkjqargv5s8qcted4jkuaqcqzysxqyz5vqsp5pz844v024ngys76xlhxywq25hep88h7z7t7y4jazj9zawqwldjys9qyyssqdl6dlne3mnkn9zpedjacz28qd3p35rwhz3w2t9mt8phu7nvdh6z4zad80efjm3snlm2q7q7p4dj56m6grc9qrn7w2kgmrn9sa3z3tsgp6k9as5

Mostro sent you this message for order id: 6d30e4eb-7ca0-4f81-9c3e-3da6ac262d8d at 20:01 date - 26/03/2024

Action: DisputeInitiatedByYou
Content: None
```

**Comment by Catrya** [comment]
Currently, mostro also allows the buyer to open the dispute without placing an invoice. Just by take a sale, that shouldn't happen

**Comment by arkanoider** [comment]
this should be fixed with #230 ?

**Comment by Catrya** [comment]
@arkanoider  yes, now with the new modification yes, it is already solved
